app/controllers/api/reservations_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class API::ReservationsController < API::BaseController
+  before_action :set_reservation, only: %i(destroy)
+
+  def index
+    beggining_of_this_month = params[:beggining_of_this_month]
+    end_of_this_month = params[:end_of_this_month]
+    @reservations = Reservation.where(date: beggining_of_this_month..end_of_this_month).includes(:user)
```

Comment:
一度しか出てこないんだったら変数化しなくていいかも

```suggestion
    @reservations = Reservation.where(
      date: params[:beggining_of_this_month]..params[:end_of_this_month]
    ).includes(:user)
```

Reply:
@komagata 
ありがとうございます！
変数化やめました